### ISSUE TITLE
Allow custom message for restricted modules

### ISSUE THREAD

**Issue by stoeffel** [ndmitchell/hlint#719]
Hey!
We would like to inform users about ways to use alternatives to restricted modules. It would be nice if you could add a custom message to hlint errors for restricted modules.
Is there a way to do this already or would that be a feature you think is worthwhile adding to hlint?
```yaml
- modules:
  - {name: Some.Banned.Module, within: [], alternative: Something.Better}
```

Thanks for your feedback

**Comment by ndmitchell** [comment]
Sounds like a great idea. I'd suggest simply "message" as the field name - sometimes you might want to explain why it's a terrible idea without having a good direct alternative. PR most welcome.

**Comment by stoeffel** [comment]
SGTM! I'll make a PR this week.
